model/cluster_utility.go

```diff
@@ -169,6 +172,76 @@ func (h *UtilityGroupVersions) AsMap() map[string]*HelmUtilityVersion {
 	}
 }
 
+// UnmarshalJSON is a custom JSON unmarshaler that can handle both the
+// old Version string type and the new type. It is entirely
+// self-contained, including types, so that it can be easily removed
+// when no more clusters exist with the old version format.
+// TODO DELETE THIS
+func (h *UtilityGroupVersions) UnmarshalJSON(bytes []byte) error {
+	type innerUtilityGroupVersions struct {
+		PrometheusOperator  *HelmUtilityVersion
+		Thanos              *HelmUtilityVersion
+		Nginx               *HelmUtilityVersion
+		NginxInternal       *HelmUtilityVersion
+		Fluentbit           *HelmUtilityVersion
+		Teleport            *HelmUtilityVersion
+		Pgbouncer           *HelmUtilityVersion
+		Promtail            *HelmUtilityVersion
+		Rtcd                *HelmUtilityVersion
+		NodeProblemDetector *HelmUtilityVersion
+		MetricsServer       *HelmUtilityVersion
+		Velero              *HelmUtilityVersion
+		Cloudprober         *HelmUtilityVersion
+	}
+
+	var versions *innerUtilityGroupVersions = &innerUtilityGroupVersions{}
+
+	err := json.Unmarshal(bytes, versions)
```

Comment:
Can we Unmarshal directly into the receiver `h`? In that case, If `err == nil`, we do not need to assign these values into  the receiver.